eole/bin/convert/convert_HF.py

```diff
@@ -23,7 +24,8 @@
 
 key_maps = {}
 key_maps["LlamaForCausalLM"] = {
-    "layer_prefix": "model.layers.",
+    "decoder_layer_prefix": "model.layers.",
+    "encoder_layer_prefix": None,
```

Comment:
`.get("encoder_layer_prefix", None)` in retrievals below might be a bit cleaner/more robust